opdracht1.js

```diff
@@ -0,0 +1,33 @@
+const tvNames = inventory.map ((tv) =>{
+    return tv.type;
+})
+
+console.log("1A; Tv types", tvNames);
+
+const soldOut = inventory.filter((tv) =>{
+    return tv.originalStock === tv.sold;
+})
+
+console.log("1B; Sold out ", soldOut);
+
+const ambiTv = inventory.filter((tv) =>{
+    return tv.options.ambiLight === true;
+})
+
+console.log("1C; Ambilight " , ambiTv);
+
+function sortTv(array) {
```

Comment:
Beschrijf functies zo goed mogelijk naar wat ze doen. Dit bevorderd de onderhoudbaarheid van je code en dan weten anderen ook direct wat ze kunnen verwachten van de functie
```suggestion
function sortTvsOnPrice(array) {
```